docs/app/Examples/elements/Button/Types/ButtonLabeledIconExample.js

```diff
@@ -1,16 +1,14 @@
 import React, { Component } from 'react'
-import { Button, Icon } from 'stardust'
+import { Button } from 'stardust'
 
 export default class ButtonLabeledIconExample extends Component {
   render() {
     return (
       <div>
-        <Button className='labeled icon'>
-          <Icon className='pause' />
+        <Button labeled='icon' icon='pause'>
           Pause
         </Button>
-        <Button className='right labeled icon'>
-          <Icon className='right arrow' />
+        <Button labeled='right icon' icon='right arrow'>
```

Comment:
### Labeled

Hm, I recall some left/right classes in SUI are order dependent.  I decided to test the classes generated by `labeled='right icon'` and indeed, this will not work :(  The classes generated will be `right icon labeled`.  The combination `right labeled` no longer appears.  Here's what happens:

***

![http://g.recordit.co/Dt8lPPCGSF.gif](http://g.recordit.co/Dt8lPPCGSF.gif)

***

Because of this, I propose we only allow `labeled` and `labeled='right'` use of the labeled prop.  This describes the label itself.  The icon then is a separate matter, I'll comment on that elsewhere so we have two threads to discuss.